### PR TITLE
Port to arch linux

### PR DESCRIPTION
**Pull request by ghost**
This is an Arch Linux port using the /opt dir as base and also using yay instead of pacman as pkg manager. Also recursively chown /opt/jok3r as your user and user_group; the scripts will now ask for sudo passwd when needed; no need to be root user. There may be a few warnings I still need to clean up but no major failures. If any arise; please message me and I will work to remedy them as soon as possible.

### CODE REVIEW

File: lib/core/Tool.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def remove(self, settings):
             logger.info('Delete RVM environment ({ruby}@{name})...'.format(
                 ruby=self.virtualenv,
                 name=self.name))
-            cmd = 'source /usr/local/rvm/scripts/rvm; rvm use {ruby} && ' \
+            cmd = 'source /home/mrgfy/.rvm/scripts/rvm; rvm use {ruby} && ' \
```
Comment (nastyagrifon) [inline lib/core/Tool.py:135]: hardcoded home path, use `$HOME`  instead

**Comment by nastyagrifon** [comment]
93 commits with no name and description, hardcoded homepath

This has to be a joke

**Comment by ghost** [comment]
i fixed it, getting used to using git as well, i guess it's a joke in hindsight, still pretty new to this whole thing

**Comment by ghost** [comment]
> 93 commits with no name and description, hardcoded homepath
> 
> This has to be a joke

still learning this, and it's my forked project so.... idk y anyone is here tbh lol

**Pull request closed without merging**